### ISSUE TITLE
Unable to play after changing Mac account username

### ISSUE THREAD

**Issue by SylvainBigonneau** [marzent/XIV-on-Mac#89]
Hi!

Ever since I changed my Mac OS account's username (and therefore, my home folder name), the game cannot launch past the character selection screen, with the usual "unable to create character config data". It is not a permission issue, as I have checked and "XIV on Mac" has full disk access.

My guess is that somewhere deep inside some cache, the game is still instructed to look into a folder from inside my old home folder: I also noticed that when trying to reset config from the "Troubleshoot" menu, I also get an error claiming that I have not given permission to the app.

The sad thing is that I thought I could fix it by re-installing the game, but to no avail! I deleted everything about XIV on Mac in `Application Support`, `Caches`, `.config` and all, but when I re-install and re-launch the app, it somehow still remembers my username as it is already filled-in in the login form!

Anybody knows how to eradicate XIV on Mac once and for all, to see if that would fix my issue?

Thanks!

**Comment by marzent** [comment]
The issue you are describing does sound very weird, although apart from the game path in the preferences file (at `~/Library/Preferences`) everything should be relative to the current user…

If I were you I would delete that one first (or adjust the game path from XOM itself) and see if that helps

**Comment by SylvainBigonneau** [comment]
Thank you, I did download those files as well, to no avail

**Comment by marzent** [comment]
You could always try AppCleaner although it will not delete more files than what you mentioned… Do you by chance have special characters in your new username?

**Comment by SylvainBigonneau** [comment]
Yes, I did use AppCleaner haha. Still, the "unable to create character config data" appears.

My username only contains letters and a digit.
